src/pgrepr/src/format.rs

```diff
@@ -7,6 +7,8 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+#![forbid(missing_docs)]
```

Comment:
Rather than pushing the `forbid` into modules, which means it won't apply by default to new modules, let's change the `forbid` at the top level to `deny`, and then stick an `allow` in only the oid module.